src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Navigation.java

```diff
@@ -87,8 +87,9 @@ public enum PageMode {
         PACKAGE,
         PREVIEW,
         RESTRICTED,
-        SERIALIZED_FORM,
         SEARCH,
+        SEARCH_TAGS,
```

Comment:
I have a sense of unease when enum constants are reordered. But it's an internal enum class, so any effects should be confined to jdk.javadoc. Thanks for making them properly ordered.